experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -76,16 +77,27 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
         @warn("Running with ρe_int in coupled mode is not tested yet.", maxlog = 1)
     end
 
+    # define variables
+    ρ_flux_h_tot = integrator.p.precomputed.sfc_conditions.ρ_flux_h_tot
+    ρ_flux_q_tot = integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot
+    ᶠradiation_flux = integrator.p.radiation.ᶠradiation_flux
+    ρ_flux_uₕ = integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ
+    col_integrated_rain = integrator.p.precipitation.col_integrated_rain
+    col_integrated_snow = integrator.p.precipitation.col_integrated_snow
+    ᶜS_ρq_tot = integrator.p.precipitation.ᶜS_ρq_tot
+    ᶜ3d_rain = integrator.p.precipitation.ᶜ3d_rain
+    ᶜ3d_snow = integrator.p.precipitation.ᶜ3d_snow
+
     # set initial fluxes to zero
-    @. integrator.p.precomputed.sfc_conditions.ρ_flux_h_tot = Geometry.Covariant3Vector(FT(0.0))
-    @. integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot = Geometry.Covariant3Vector(FT(0.0))
-    @. integrator.p.radiation.ᶠradiation_flux = Geometry.WVector(FT(0))
-    integrator.p.precomputed.sfc_conditions.ρ_flux_uₕ.components .= Ref(SMatrix{1, 2}([FT(0), FT(0)]))
-    integrator.p.precipitation.col_integrated_rain .= FT(0)
-    integrator.p.precipitation.col_integrated_snow .= FT(0)
-    integrator.p.precipitation.ᶜS_ρq_tot .= FT(0)
-    integrator.p.precipitation.ᶜ3d_rain .= FT(0)
-    integrator.p.precipitation.ᶜ3d_snow .= FT(0)
+    @. ρ_flux_h_tot = Geometry.Covariant3Vector(FT(0.0))
+    @. ρ_flux_q_tot = Geometry.Covariant3Vector(FT(0.0))
+    @. ᶠradiation_flux = Geometry.WVector(FT(0))
+    ρ_flux_uₕ.components .= Ref(SMatrix{1, 2}([FT(0), FT(0)]))
+    col_integrated_rain .= FT(0)
```

Comment:
You could use the `@.` broadcasting macro everywhere, but that's just aesthetics. :)